nx 22.7.0 doesn't seem to work on Windows

### Current Behavior

Hi, after updating global nx from 22.5.3 to 22.7.0, some nx comands fail with the following error:

Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'd:'

I keep both the monorepo and PNPM store (not to be confused with node_modules) on the D: drive.

Running `nx version` command works, both within and outside the monorepo. `nx report` runs outside the monorepo, but running it within causes the same error to appear.

### Expected Behavior

Nx commands shouldn't fail

### GitHub Repo

_No response_

### Steps to Reproduce

1. Use pnpm and the latest version of global nx
2. Open an nx workspace on a D: drive
3. Run `nx report`

### Nx Report

```shell
PS D:\code\monorepo> nx report             
Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'd:'
    at throwIfUnsupportedURLScheme (node:internal/modules/esm/load:195:11)
    at defaultLoadSync (node:internal/modules/esm/load:142:3)
    at #loadAndMaybeBlockOnLoaderThread (node:internal/modules/esm/loader:795:12)
    at #loadSync (node:internal/modules/esm/loader:815:49)
    at ModuleLoader.load (node:internal/modules/esm/loader:780:26)
    at ModuleLoader.loadAndTranslate (node:internal/modules/esm/loader:526:31)
    at #getOrCreateModuleJobAfterResolve (node:internal/modules/esm/loader:571:36)
    at afterResolve (node:internal/modules/esm/loader:624:52)
    at ModuleLoader.getOrCreateModuleJob (node:internal/modules/esm/loader:630:12)
    at onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:649:32) {
  code: 'ERR_UNSUPPORTED_ESM_URL_SCHEME'
}
```

### Failure Logs

```shell
Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'd:'
    at throwIfUnsupportedURLScheme (node:internal/modules/esm/load:195:11)
    at defaultLoadSync (node:internal/modules/esm/load:142:3)
    at #loadAndMaybeBlockOnLoaderThread (node:internal/modules/esm/loader:795:12)
    at #loadSync (node:internal/modules/esm/loader:815:49)
    at ModuleLoader.load (node:internal/modules/esm/loader:780:26)
    at ModuleLoader.loadAndTranslate (node:internal/modules/esm/loader:526:31)
    at #getOrCreateModuleJobAfterResolve (node:internal/modules/esm/loader:571:36)
    at afterResolve (node:internal/modules/esm/loader:624:52)
    at ModuleLoader.getOrCreateModuleJob (node:internal/modules/esm/loader:630:12)
    at onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:649:32) {
  code: 'ERR_UNSUPPORTED_ESM_URL_SCHEME'
}
```

### Package Manager Version

pnpm 10.33.0

### Operating System

- [ ] macOS
- [ ] Linux
- [x] Windows
- [ ] Other (Please specify)

### Additional Information

_No response_

I do have `error TS6059: File 'C:/dev/app/packages/backend-nest-util-common/interfaces/src/start-date.interface.ts' is not under 'rootDir' '/dev/app/packages/backend-nest-util-common'` issues since the update to version 22.7.0. My `tsconfig.lib.json` file looks like:

```
{
	"extends": "./tsconfig.json",
	"compilerOptions": {
		"outDir": "../../dist/out-tsc",
		"declaration": true,
		"types": ["node"],
		"target": "es2021",
		"experimentalDecorators": true,
		"emitDecoratorMetadata": true,
		"strictNullChecks": true,
		"noImplicitAny": true,
		"strictBindCallApply": true,
		"forceConsistentCasingInFileNames": true,
		"noFallthroughCasesInSwitch": true
	},
	"include": ["**/*.ts"],
	"exclude": ["jest.config.ts", "**/*.spec.ts", "**/*.test.ts"]
}

```

`include` includes all TypeScript files because I do have sub packages within the package and my files do not live only in the `/src` folder.

I use Windows 11.

--- a/packages/nx/bin/nx.ts
+++ b/packages/nx/bin/nx.ts
@@ -146,9 +146,9 @@ async function main() {
       warnIfUsingOutdatedGlobalInstall(GLOBAL_NX_VERSION, LOCAL_NX_VERSION);
       if (localNx.includes('.nx')) {
         const nxWrapperPath = localNx.replace(/\.nx.*/, '.nx/') + 'nxw.js';
-        await import(nxWrapperPath);
+        require(nxWrapperPath);
       } else {
-        await import(localNx);
+        require(localNx);
       }
     }
   }

